src/test/java/pl/edu/icm/pl/mxrdr/extension/importer/cif/file/internal/CifMetadataReaderTest.java

```diff
@@ -0,0 +1,99 @@
+package pl.edu.icm.pl.mxrdr.extension.importer.cif.file.internal;
+
+
+import org.apache.commons.lang3.StringUtils;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.rcsb.cif.model.StrColumn;
+import org.rcsb.cif.schema.mm.Entry;
+import org.rcsb.cif.schema.mm.MmCifBlock;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.empty;
+import static pl.edu.icm.pl.mxrdr.extension.importer.cif.file.internal.CifFileImporterTestHelper.collectToList;
+
+@ExtendWith(MockitoExtension.class)
+class CifMetadataReaderTest {
+
+    private static final String VALUE = "A";
+
+    @Mock
+    private MmCifBlock mainBlock;
```

Comment:
Easier to parse a real file than mock a value object. Also provides better context than mocking random method calls that makes that part of implementation hard-coded in the test via mocking.